### DISCUSSION TITLE
deploying Jupyter Lite internal LAN network

### DISCUSSION THREAD

**Discussion by bugsbunny88** [jupyterlite/jupyterlite#1253] [Q&A] [answered]
I have a single server set up at home with a couple of web services all running neatly behind a traefik reverse proxy. I was looking for a way to distribute a jupyterlite website via a docker-compose files ; was wondering if this was possible. I have recently learned how to generate my own image using buildx , if I have a Dockerfile with the proper instruction , what would be helpful is if there is a step that can bake in a simple server to serve the necessary assets. If I could see an example that I could work off would be very helpful.

**Comment by JackYinpei** [comment, marked as answer]
The author recommends using jupyter lite for deployment, and I also think this method is quite simple. 

The folder generated by `jupyter lite build` can be served directly. This is a static service, so you can completely proxy this folder directly with nginx, and then you'll be able to access it. 

By the way, I didn't find a way to deploy with docker either. If you happen to find one, please let me know.
